frontend/src/components/housing/submission/SubmissionForm.vue

```diff
@@ -723,8 +718,8 @@ onMounted(async () => {
     </div>
     <ATSUserLinkModal
       v-model:visible="atsUserLinkModalVisible"
-      :f-name="values.contactFirstName"
-      :l-name="values.contactLastName"
+      :f-name="values.contacts[0]?.firstName ?? ''"
+      :l-name="values.contacts[0]?.lastName ?? ''"
```

Comment:
Style: Is there a reason why the attributes here are named `l-name` and `f-name` instead of a full identification? May be worth revisiting for consistency purposes as a single letter can lead to definition ambiguity.

Reply:
There are state values by `firstName` and `lastName` which is probably what led to it. But i've renamed the props for more clarity.